src/video/emscripten/SDL_emscriptenmouse.c

```diff
@@ -215,7 +239,7 @@ Emscripten_ShowCursor(SDL_Cursor* cursor)
             }
         }
         else {
-            EM_ASM(
+            MAIN_THREAD_EM_ASM_INT(
```

Comment:
This seems to have gained an extra `_INT`.

Reply:
removed, seems the `_INT` was removed since the old PR. Thanks for the review